cmd/bedMaxWig/bedMaxWig.go

```diff
@@ -12,14 +12,27 @@ import (
 	"log"
 )
 
-func bedMaxWig(infile string, database string, chromsizeFile string, outfile string) {
+func bedMaxWig(infile string, database string, chromsizeFile string, outfile string, norm bool) {
 	var records []bed.Bed = bed.Read(infile)
 	var wigData []wig.Wig = wig.Read(database)
 	var sizes []chromInfo.ChromInfo = chromInfo.ReadToSlice(chromsizeFile)
 	var outlist []bed.Bed
 	var currentBed bed.Bed = records[0]
 	var chromSlice []float64
 	var i int
+	var wigTotal float64 = 0
+
+	if norm == true { //TODO: This assumes fixedStep. Either should check for fixedStep or be able to handle both cases (fixedStep and variableStep)
+		for i := 0; i < len(wigData); i++ { //Goal here is to cycle through all the "chromosomes" of the wig: wig[0], wig[1], etc.
```

Comment:
Just a style comment. This could be `for i := range wigData { do stuff with wigData[i] }`. Ranges are generally easier to read if you are iterating over all elements in a slice. The exception is when `i` does not start at 0 in which case you need a traditional for loop.

Reply:
If consensus for the lab is to use range, I'm happy to change it and refresh this PR.

Reply:
I think Dan makes a good point that it is a little quicker to read and possibly less prone to bugs to use the "range" syntax when doing a "normal" for loop over all elements of a slice.  Not a big deal, but something to keep in mind.